src/frontend/citizen-portal/src/app/components/find-ticket/find-ticket.component.ts

```diff
@@ -90,7 +90,8 @@ export class FindTicketComponent implements OnInit {
     const data: DialogOptions = {
       titleKey: "Upload Ticket",
       actionType: "primary",
-      messageKey: `If you cannot upload a copy of your handwritten ticket, you can initiate a dispute by contacting the Violation Ticket Centre: violationticketcentre@gov.bc.ca.  You must include your Given Name, Surname, Driver's Licence Number, Violation Ticket Number, and Violation Time. Do not include any evidence, such as videos or pictures, with your email. Evidence can only be presented at the hearing. `,
+      messageKey: `If you cannot upload a copy of your handwritten ticket, please contact support at Courts.TCO@gov.bc.ca.
```

Comment:
@Sharkww  , does it make sense to use` <a href="mailto:Courts.TCO@gov.bc.ca">Courts.TCO@gov.bc.ca</a>` here?

In other locations, we also include a mail icon, ie.
`<a href="mailto:Courts.TCO@gov.bc.ca" target="blank">Courts.TCO@gov.bc.ca <img class="me-2" src="/assets/mail-icon.svg" style="width: 15px; height: 13px; vertical-align: text-top;"/></a>`

Is there a standard or best practice?

Reply:
messageKey of Dialog may not able to support html code

Reply:
The mail icon mainly base on the design, not a standard or best practice in this case

Reply:
So is this okay to merge in now?

Reply:
Yes, I think we're good to go then. Great work.

Reply:
@karahanb I prefer having mailto link, you can replace `{{ options.messageKey | translate }}` with `{{{ options.messageKey | translate }}}` and test it. But please also take a look to other dialog that using `messageKey` will be affected or not. Generally, I think it will be fine.